build_local.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -e
+function usage() {
+	cat <<EOF
+Usage: $0
+    [--version]
+
+required arguments:
+    --version      Specify the version for image
+EOF
+	exit 1
+}
+
+
+function handle_parameters() {
+    version=""
+    until [ -z $1 ]; do
+        case $1 in
+        --version )
+            version="$2"
```

Comment:
This can lead to UB in case that there is flag `--version` but there is no value for it.
consider adding this validation:
```
if [ -n "$2" ]; then
    version="$2"
fi
```

Reply:
If no value you will get usage text:
version is missing
Usage: ./build_local.sh
    [--version]

required arguments:
    --version      Specify the version for image